library/portable-simd/crates/core_simd/src/mod.rs

```diff
@@ -35,6 +35,5 @@ pub mod simd {
     pub use crate::core_simd::masks::*;
     pub use crate::core_simd::ord::*;
     pub use crate::core_simd::swizzle::*;
-    pub use crate::core_simd::swizzle_dyn::*;
```

Comment:
Delete this line as [﻿swizzle_dyn.rs](https://github.com/rust-lang/rust/blob/master/library/portable-simd/crates/core_simd/src/swizzle_dyn.rs) exports nothing.